tests/autoyast/installation.pm

```diff
@@ -267,6 +268,13 @@ sub run {
             wait_screen_change { send_key 'alt-u' };
             next;
         }
+	elsif (match_has_tag('salt-formula-motd-setup')) {
```

Comment:
Hnn, looks like you broke indentation, feels like it tabs vs spaces issue here.

Reply:
i know. i havent run the tidy on it. i was expecting the ci. i will fix it